pinot-integration-tests/src/test/java/org/apache/pinot/integration/tests/BaseRealtimeClusterIntegrationTest.java

```diff
@@ -92,9 +92,11 @@ protected void overrideServerConf(PinotConfiguration configuration) {
   protected List<String> getNoDictionaryColumns() {
     // Randomly set time column as no dictionary column.
     if (new Random().nextInt(2) == 0) {
-      return Arrays.asList("ActualElapsedTime", "ArrDelay", "DepDelay", "CRSDepTime", "DaysSinceEpoch");
+      return Arrays.asList("ActualElapsedTime", "ArrDelay", "DepDelay", "CRSDepTime", "RandomAirports",
```

Comment:
We can make it more readable by
```suggestion
      List<String> noDictionaryColumns = new ArrayList<>(super.getNoDictionaryColumns());
      noDictionaryColumns.add("DaysSinceEpoch");
      return noDictionaryColumns;
```

Reply:
Done.